app/assets/javascripts/templates.js

```diff
@@ -0,0 +1,4 @@
+window.templates = {};
```

Comment:
I don't think this file should be comitted...?

Reply:
Thanks for spotting. It's populated by a rake task and found its way into a commit.